pkg/operator/defaultstorageclass/controller.go

```diff
@@ -196,6 +196,10 @@ func newStorageClassForCluster(infrastructure *configv1.Infrastructure) (*storag
 		storageClassFile = "storageclasses/gcp.yaml"
 	case configv1.VSpherePlatformType:
 		storageClassFile = "storageclasses/vsphere.yaml"
+	case configv1.AzurePlatformType:
```

Comment:
Seems the unit test also need to adapt the change.
Need revert: https://github.com/openshift/cluster-storage-operator/commit/0d93ba2a28faafe0f4cd587c3e48f45a83cab961

Reply:
Indeed, reverted.